CriticalMass/SettingsSection.swift

```diff
@@ -23,11 +23,13 @@ enum Section: CaseIterable {
         var title: String?
         var subtitle: String?
         var action: Action
+        let accessibilityIdentifier: String
```

Comment:
Minor: can we make them all var or all let?

Reply:
Good idea, they shouldn't change after initialization, so I decided to make them all `let`s :)